### ISSUE TITLE
Endless backend loading on element creation (empty template)

### ISSUE THREAD

**Issue by johannessteu** [johannessteu/neos-development-collection#769]
Jira issue originally created by user :

As I created a new NodeType and the corresponding Fluid Template that should output the content I got an endless backend loading as I wanted to create the element in the backend.

This is the code in the Fluid template:
<f:if condition="{content}">
    {content -> f:format.raw()}
/f:if

So if the Template content is empty it causes the backend to load endlessly on element creation.

 Atachements:

![Jira Image](https://jira.neos.io/secure/attachment/12611/Screenshot.PNG) 

 Jira-URL: https://jira.neos.io/browse/NEOS-136

**Comment by johannessteu** [comment]
Comment created by @aertmann:

Probably due to the content element wrapping service..

**Comment by johannessteu** [comment]
Comment created by kerstin_gerull:

Hi,

I have the same issue but with content elements with existig markup.
When I add a content element with the plus (+) button on the "Existing Element" the spinner is never removed and the new content element is not displayed. (see Screenshot). There is no JS error or something else visible.
But the content element was added, when you reload the page you see it.

Edit: my neos version is 1.2.13.

Greetings Kerstin

**Comment by johannessteu** [comment]
Comment created by kerstin_gerull:

Hi again,

I have debugged a little.

The TS path was something like this:

```
"page<Test:Page>/body<TYPO3.TypoScript:Template>/content/main<TYPO3.Neos:PrimaryContent>/default<TYPO3.TypoScript:Matcher>/element<TYPO3.Neos:ContentCollection>/**meta/process/contentElementWrapping/expression<TYPO3.Neos:ContentElementWrapping>"
```

Like Aske said the ContentElementWrapping is attached here and it is not possible to remove it, because the backend would not work.

I ended up in the function "TYPO3\TypoScript\Core\Runtime -> evaluateInternal()".

There is an if clause which is checking if there is a meta process, which would be the contentElementWrapping.
But the $output is empty before and after, so the contentElementWrapping is not deleting the content.

```
Line 461: if (isset($typoScriptConfiguration['**meta']['process'])) {}
```

Because of this I was checking if the $output is empty before this and found the following lines.
The $evaluateObject is TRUE and the return of $tsObject->evaluate() ist NULL.

```
Line 442ff:
if ($evaluateObject) {
    $output = $tsObject->evaluate();
} else {...}
```

I could not go further, but maybe that is a start for someone else.

Greetings Kerstin
